packages/syntax-core/src/Tabs/TabInternal.tsx

```diff
@@ -0,0 +1,106 @@
+import { type ComponentProps, type ReactElement, type ReactNode } from "react";
+import classnames from "classnames";
+import Box from "../Box/Box";
+import Typography from "../Typography/Typography";
+import styles from "./TabInternal.module.css";
+
+const ItemCountIcon = ({
+  itemCount,
+  on,
+  selected,
+}: {
+  itemCount: number;
+  on: ComponentProps<typeof TabInternal>["on"];
+  selected: boolean;
+}) => {
+  const unselectedItemCountStyle =
+    on === "lightBackground"
+      ? styles.unselectedItemCount
+      : styles.unselectedItemCountOnDarkBackground;
+  const selectedItemCountStyle =
+    on === "lightBackground"
+      ? styles.selectedItemCount
+      : styles.selectedItemCountOnDarkBackground;
+  const itemCountStyles = classnames(styles.itemCount, {
+    [unselectedItemCountStyle]: !selected,
+    [selectedItemCountStyle]: selected,
+  });
+  return (
+    <div className={itemCountStyles}>
```

Comment:
a little inclined to replace all of these styles with just a syntax Box and have a getBackground or something util to tweak your stylings.

if not, then i think at least wrap the outside with a <Box/> so you dont have to do the manual this stuff

```
.itemCount {
  border-radius: 999px;
  display: flex;
  align-items: center;
  justify-content: center;
  padding: 4px;
}
```